src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs

```
@@ -350,17 +350,7 @@ internal virtual bool IsPatternSupported(UiaCore.UIA patternId)
             return false;
         }
 
-        internal virtual int[] RuntimeId
-        {
-            get
-            {
-                string message = string.Format(SR.AccessibleObjectRuntimeIdNotSupported, nameof(AccessibleObject), nameof(RuntimeId));
-
-                Debug.Fail(message);
-
-                throw new NotSupportedException(message);
-            }
-        }
+        internal virtual int[] RuntimeId => Array.Empty<int>();
```

If we are not returning a valid ID, we should throw an exception according to docs - https://docs.microsoft.com/en-us/windows/win32/api/uiautomationcore/nf-uiautomationcore-iaccessibleex-getruntimeid. Probably with Exception.HResult property set to E_NOTIMPL. 
At least it would be good to understand how an empty array is processed by the UIA.